### ISSUE TITLE
The PickedItem component does not show up nicely

### ISSUE THREAD

**Issue by saltnpixels** [10up/block-components#34]
**Describe the bug**

The contentPicker has some stylistic issues when used in inspector controls. It also has some html issues.

1. The class `.block-editor-link-control__search-item-title` is set to have white-space: nowrap; This causes the item to go offscreen and the "X" cannot be reached to remove the post from the list. 
2. The items are being decoded on line 80 in PickedItem.js. if html is used in the title, that comes through decoded. Probably better to remove any html

** Fix **

In PickedItem.js on line 80, maybe we can remove the entities completely:
```JS
decodeEntities(suggestion.title)
// to
decodeEntities(suggestion.title).replace(/(<([^>]+)>)/gi, "")
```

The same would be done in SearchItem.js on line 38.

As for the CSS that would need to be scoped to the Inspector.

**Screenshots**

<img width="316" alt="Screen Shot 2021-07-13 at 4 17 56 PM" src="https://user-images.githubusercontent.com/11615474/126365850-c1eb60c2-178a-4581-b7f9-5f8ad9fbab57.png">

**Comment by tlovett1** [comment]
Can you open up a PR?

**Comment by xavortm** [comment]
I checked this on develop; now [it looks like this](https://i.imgur.com/OiNo8UU.png)

Tested in this setup:
```jsx
<InspectorControls>
	<PanelBody>
		<ContentPicker
			onPickChange={(pickedContent) => {
				console.log(pickedContent);
			}}
			mode="post"
			label="Please select a Post or Page:"
			contentTypes={['post', 'page']}
		/>
	</PanelBody>
</InspectorControls>
```

I quickly checked out an older commit (before the reported issue in develop) and it is still working nicely. Would it be some browser support issue? I think the screenshot is from Chrome however...

The `block-editor-link-control__search-item-title` element has `overflow: hidden; text-overflow: ellipsis;` set to it.

No-wrap is working nicely with the text-overflow here. If it is removed, the full title can be made visible (maybe a good idea) but it will go on [multiple lines then like this](https://cln.sh/pgvTr9745oqWSgAq1xAj).

**Comment by Firestorm980** [comment]
Just doing some additional research on this one, it looks like it may have been resolved in Gutenberg via https://github.com/WordPress/gutenberg/issues/33586#issuecomment-888921188. When I checked in a local 5.9.1 install, title text was wrapping (I think this was changed with 5.9).

As for stripping the HTML, we could put something in, but I get the feeling that it was project specific. When I pulled down the most recent code on it, it was no longer an issue due to some added conditionals checking for REST requests from the filter that introduced that additional markup. If we're OK with that being a project specific bug, we may be able to close.
